deploy/gk-deploy

```diff
@@ -546,6 +548,10 @@ fi
 eval_output "${CLI} create -f heketi-storage.json 2>&1"
 
 check_pods "job-name=heketi-storage-copy-job" "Completed"
+if [[ ${?} != 0 ]]; then
+  output "Error waiting for copy-job to complete."
```

Comment:
Explicitly call it `job heketi-storage-copy-job` so folks can inspect it after the exit.